### PR TITLE
Feature/oph24 13 nav bar

### PR DESCRIPTION
**Pull request by PwFaze**
## Change made

- [ ] Bug fixes
- [x] New features
- [ ] Breaking changes

## Describe what you have done

-make nav bar responsive and create footer

### New Features

-nav bar is now responsive
-footer is available

### Fix

-

### Others

-

### CODE REVIEW

File: src/components/Credit.astro
Comment (shalluv) [inline src/components/Credit.astro:1]: SVGs take too much space. Please move it to a new file.

File: src/components/Nav.astro
Comment (shalluv) [inline src/components/Nav.astro:1]: I think this code is too complex for its purpose. React might be the better way to go!

**Review by shalluv — CHANGES_REQUESTED** [review]

File: src/layouts/BaseLayout.astro [hunk 1]
```diff
@@ -4,7 +4,7 @@ import "@fontsource/ibm-plex-sans-thai/500.css";
 import "@fontsource/ibm-plex-sans-thai/600.css";
 import "@fontsource/ibm-plex-sans-thai/700.css";
 
-import Nav from "@/components/Nav.astro";
+import NavBar from "@/components/Nav.tsx";
```
Comment (ImSoZRious) [inline src/layouts/BaseLayout.astro:7]: I don't think `.tsx` is nescessary.
```suggestion
import NavBar from "@/components/Nav";
```

**Review by ImSoZRious — COMMENTED** [review]
👍

**Review by shalluv — APPROVED** [review]

**Pull request merged**